<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tv_flights"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_departure">
          <property name="text">
           <string>Вылет</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_arrival">
          <property name="text">
           <string>Прилёт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_isDate">
          <property name="text">
           <string>Выбрать дату?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="de_SelectDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumDateTime">
           <datetime>
            <hour>20</hour>
            <minute>55</minute>
            <second>0</second>
            <year>2017</year>
            <month>9</month>
            <day>14</day>
           </datetime>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>2</hour>
            <minute>45</minute>
            <second>0</second>
            <year>2016</year>
            <month>8</month>
            <day>15</day>
           </datetime>
          </property>
          <property name="maximumDate">
           <date>
            <year>2017</year>
            <month>9</month>
            <day>14</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2016</year>
            <month>8</month>
            <day>15</day>
           </date>
          </property>
          <property name="maximumTime">
           <time>
            <hour>20</hour>
            <minute>55</minute>
            <second>0</second>
           </time>
          </property>
          <property name="minimumTime">
           <time>
            <hour>2</hour>
            <minute>45</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_Aeroports"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ShowStat">
          <property name="text">
           <string>Статистика</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ShowTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Таблица рейсов</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
